charts/primary-site/values.yaml

```
@@ -19,9 +19,9 @@ globals:
     storageProvider: google_cloud
     bucketName: foxglove-inbox
   
+  ## Optionally used to configure http/https proxy egress
```

```suggestion
  ## Configure an http/https egress proxy if your environment requires outgoing connections to go through a proxy. Set `enabled` to `true` and one of `httpProxy` or `httpsProxy`
```

I don't know if this wording is exactly right but I would put something like this. What is the purpose of `noProxy`?

---

`NO_PROXY` is a list of hosts to not proxy for

---

Updated